src/tools/pub/cmd/index.md

```diff
@@ -12,7 +12,7 @@ Flutter has its own commands for managing and updating packages.
 For more information, see
 [Using Packages]({{site.flutter}}/using-packages/) and
 [Upgrading Flutter]({{site.flutter}}/upgrading/)
-on the [Flutter website]({{site.flutter}}).
+on the [Flutter website.]({{site.flutter}})
```

Comment:
Tiny nit: it seem like the period outside the link is the more common style in this repo.

Reply:
It is, but it looks bad due to the little "external link" icon. I've been fixing them as I find them.